src/Servers/Kestrel/Core/src/Features/WebTransportStreamType.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Features;
+
+/// <summary>
+/// Represents the different types of WebTransport streams.
+/// </summary>
+internal enum WebTransportStreamType
```

Comment:
If this isn't used with IWebTransportSession then it should be moved to a different folder.